devnotes/pnor_layout.md

```diff
@@ -0,0 +1,161 @@
+| DJVPD                       | 06 | 000E5000:0012CFFF  | 48000   | d    | -     | same as size | _unknown_                   |
+| MVPD                        | 07 | 0012D000:001BCFFF  | 90000   | d    | -     | same as size | _unknown_                   |
+| CVPD                        | 08 | 001BD000:00204FFF  | 48000   | d    | -     | same as size | _unknown_                   |
+| [HBB](#HBB)                 | 09 | 00205000:00304FFF  | 100000  | d    | -     | same as size | bootblock                   |
+| HBD                         | 0A | 00305000:00424FFF  | 120000  | d    | -     | same as size | _unknown_                   |
+| [HBI](#HBI)                 | 0B | 00425000:019C4FFF  | 15A0000 | d    | -     | same as size | romstage, ramstage, payload |
+| SBE                         | 0C | 019C5000:01A80FFF  | BC000   | d    | -     | same as size | _unknown_                   |
+| [HCODE](#HCODE)             | 0D | 01A81000:01BA0FFF  | 120000  | d    | -     | same as size | _unknown_                   |
+| HBRT                        | 0E | 01BA1000:021A0FFF  | 600000  | d    | -     | same as size | _unknown_                   |
+| [PAYLOAD](#PAYLOAD)         | 0F | 021A1000:022A0FFF  | 100000  | d    | -     | same as size | _unknown_                   |
+| [BOOTKERNEL](#BOOTKERNEL)   | 10 | 022A1000:03820FFF  | 1580000 | d    | -     | same as size | _unknown_                   |
+| OCC                         | 11 | 03821000:03940FFF  | 120000  | d    | -     | same as size | _unknown_                   |
+| FIRDATA                     | 12 | 03941000:03943FFF  | 3000    | d    | -     | same as size | _unknown_                   |
+| [VERSION](#VERSION)         | 13 | 03944000:03945FFF  | 2000    | d    | -     | same as size | _unknown_                   |
+| BMC_INV                     | 14 | 03968000:03970FFF  | 9000    | d    | -     | same as size | _unknown_                   |
+| [HBBL](#HBBL)               | 15 | 03971000:03977FFF  | 7000    | d    | -     | same as size | _unknown_                   |
+| ATTR_TMP                    | 16 | 03978000:0397FFFF  | 8000    | d    | -     | same as size | _unknown_                   |
+| ATTR_PERM                   | 17 | 03980000:03987FFF  | 8000    | d    | -     | same as size | _unknown_                   |
+| [IMA_CATALOG](#IMA_CATALOG) | 18 | 03989000:039C8FFF  | 40000   | d    | -     | same as size | _unknown_                   |
+| RINGOVD                     | 19 | 039C9000:039E8FFF  | 20000   | d    | -     | same as size | _unknown_                   |
+| WOFDATA                     | 1A | 039E9000:03CE8FFF  | 300000  | d    | -     | same as size | _unknown_                   |
+| HB_VOLATILE                 | 1B | 03CE9000:03CEDFFF  | 5000    | d    | -     | same as size | _unknown_                   |
+| MEMD                        | 1C | 03cee000:03CFBFFF  | E000    | d    | -     | same as size | _unknown_                   |
```

Comment:
@IgorBagnucki There's a lot of `unknown` for coreboot. I think too much. We have written a substantial amount of coreboot code, so we should be able to tell if we use given partition or not. If it is currently unused (but we are not sure if we need it in the future) let's mark it as `currently unused`. Try to have as little unknows as possible.

For example , this one holds the VPD for RAM IIRC. So  we pobably already use it in coreboot? It also lacks description.

Reply:
I have verified each partition and marked which ones are required and which are fine to be erased. It is however possible that some of them are still required in later stages or as some kind of storage mid-booting.